source/routing/raptor.cpp

```diff
@@ -692,10 +692,9 @@ void RAPTOR::raptor_loop(Visitor visitor,
                     // journey pattern point before
                     const DateTime previous_dt = prec_labels.dt_transfer(jpp.sp_idx);
                     if (prec_labels.transfer_is_initialized(jpp.sp_idx) && valid_stop_points[jpp.sp_idx.val] &&
-                        (!is_onboard || visitor.better_or_equal(previous_dt, workingDt, *it_st))) {
+                        (!is_onboard || visitor.better_or_equal(previous_dt, workingDt, *it_st, shift))) {
```

Comment:
Because better_or_equal calls section_end which was one of the function shifting. But I guess I could do
```
visitor.better_or_equal(previous_dt - shift, workingDt - shift, *it_st)))
```
With the same result (I we keep the shifting ;)). I don't know why I didn't thought of this.